emulator/accountlinking_test.go

```diff
@@ -1,120 +0,0 @@
-/*
- * Flow Emulator
- *
- * Copyright Dapper Labs, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package emulator_test
-
-import (
-	"context"
-	"testing"
-
-	"github.com/onflow/cadence"
-	"github.com/onflow/cadence/runtime/common"
-	"github.com/rs/zerolog"
-
-	"github.com/onflow/flow-emulator/adapters"
-	"github.com/onflow/flow-emulator/emulator"
-
-	flowgo "github.com/onflow/flow-go/model/flow"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
-	flowsdk "github.com/onflow/flow-go-sdk"
-)
-
-func TestAccountLinking(t *testing.T) {
```

Comment:
This was testing the account-linking enabling/disabling using the pragma. Removed this test since this is no longer valid.